pom.xml

```diff
@@ -97,28 +97,23 @@
     <!-- Maven plugins -->
     <compiler-plugin.version>3.10.1</compiler-plugin.version>
     <maven.checkstyle.version>3.2.0</maven.checkstyle.version>
-    <surefire-plugin.version>3.0.0-M7</surefire-plugin.version>
+    <surefire-plugin.version>3.5.2</surefire-plugin.version>
 
     <!-- Project options -->
     <maven.compiler.parameters>true</maven.compiler.parameters>
     <maven.compiler.source>17</maven.compiler.source>
     <maven.compiler.target>17</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    
+
     <!-- Quarkus -->
-    <quarkus-plugin.version>3.15.3</quarkus-plugin.version>
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
-    <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
-    <quarkus.platform.version>3.15.3.1</quarkus.platform.version>
+    <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
+    <quarkus.platform.version>3.20.0</quarkus.platform.version>
 
     <!-- Other versions - used in Test -->
-    <log4j.version>2.17.2</log4j.version>
-    <bouncycastle.version>1.78.1</bouncycastle.version>
```

Comment:
I don't think this is managed by the Quarkus BOM (or it did not used to be). BouncyCastle is used by our tests. So having the version here is useful for overriding CVEs reported by scanners etc. So maybe we should keep it?

Reply:
It's managed by Quarkus - the version in the 3.20 BOM is 1.80. I'm fine either way, I noticed it as an IDE warning that a "managed dependency with version override" (or similar). It's nice to maintain one less thing and if overriding for a CVE someone needs to touch the pom.xml anyway. Like I said, either way is fine - let me know what's preferred.

Reply:
So ...
* Does this override the BOM? Or is it useless?
* What does Quarkus use Bouncycastle for? Bouncycastle in production code is a nogo for many users.

Reply:
> Does this override the BOM? Or is it useless?

The property by itself or the property + specifying it on the BC dependencies?

> What does Quarkus use Bouncycastle for? Bouncycastle in production code is a nogo for many users.

From what I can tell, Quarkus has the capability to register alternate security providers [1]. Without the BC dependencies in compile/runtime scope, none of that should impact the drain-cleaner. Here it's really just about using the same version that Quarkus has in its BOM.

[1] https://quarkus.io/guides/security-customization#registering-security-providers

Reply:
> > Does this override the BOM? Or is it useless?
> 
> The property by itself or the property + specifying it on the BC dependencies?

If you revert the Bouncycastle related changes you did -> what Bouncycastle version will it use?

Reply:
> If you revert the Bouncycastle related changes you did -> what Bouncycastle version will it use?

1.78.1

Reply:
I would probably keep the variable and keep using it. just align the version it is set to. I think it might be handy again one day. WDYT @im-konge?

Reply:
Yeah I wanted to say the same thing, IIRC previously there was some CVE in Bouncycastle and we got some reports that we should fix it (even if it's only used in STs). So I would keep it as it was and I would bump the version of it to the one Quarkus uses in 3.20 (or maybe even newer if that's the case).

Reply:
Bouncycastle property restored and set to 1.80 to match Quarkus. This is also the latest BC release.